src/Symfony/Component/HttpClient/CurlHttpClient.php

```diff
@@ -110,18 +108,18 @@ public function request(string $method, string $url, array $options = []): Respo
             ];
 
             if ('GET' === $method && !$options['body'] && $expectedHeaders === $pushedHeaders) {
-                $this->logger && $this->logger->info(sprintf('Connecting request to pushed response: %s %s', $method, $url));
+                $this->logger && $this->logger->debug(sprintf('Connecting request to pushed response: "%s" "%s"', $method, $url));
```

Comment:
`"%s %s"`? (everywhere in this PR)

Reply:
@nicolas-grekas It will be more difficult to spot an empty method for example.  WDYT ?

Reply:
actually, I would forbid empty method in HttpClientTrait::prepareRequest, can you add the check in this PR?

Reply:
You did a good good. HttpClientTrait::prepareRequest already forbid it :+1: